### PR TITLE
net-analyzer/bro: solving #600266

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.31, Repoman-2.3.9

### CODE REVIEW

**Comment by thinrope** [comment]
I guess you need to add the following line upon running `git commit --amend`:
Closes: https://bugs.gentoo.org/600266
This will link the PR automagically to Bugzilla (after `git push` of course)
Also it is better to keep the title more descriptive, e.g. "net-analyzer/bro-2.4.1: fix invalid python shebang"

File: net-analyzer/bro/bro-2.4.1-r2.ebuild [hunk 1]
```diff
@@ -44,6 +44,12 @@ pkg_setup() {
 	use python && python-single-r1_pkg_setup
 }
 
+src_prepare() {
+	# Disable shell script feature which sets interpreter to build time default
+	grep -rl 'set(_regex "^#!\[ \]\*/usr/bin/env' . | xargs sed -i -e '/set(_regex \"^#!\[ \]\*\/usr\/bin\/env/,+17d' || die
```
Comment (mgorny) [inline net-analyzer/bro/bro-2.4.1-r2.ebuild]: Please don't:
1. use `xargs`,
2. do `sed` with ranges that aren't clearly defined.

This thing really asks for accidental mis-sed on version bump. So, a patch would probably be better, unless you can figure out another way of doing this.

Also, if the result of this is unversioned `python` shebang, then this is wrong and you should fix it to store correct path instead.

**Comment by TheChymera** [comment]
@mgorny good?

**Comment by mgorny** [comment]
Thanks. Will test and let you know.

**Pull request closed without merging**